pom.xml

```diff
@@ -20,8 +20,8 @@
              this artifact is downloaded directly from github -->
 
         <dependency>
-            <groupId>com.github.ziccardi</groupId>
-            <artifactId>aerogear-push-helper</artifactId>
+            <groupId>at.ftec.aerogear</groupId>
```

Comment:
So, I think, once the other PR is merged (and lib released?) we need to go back to the github org